### ISSUE TITLE
Change of behaviour from lightning logger

### ISSUE THREAD

**Issue by cgoo4** [awslabs/gluon-ts#3087]
## Description
A torch deepar model I'm training had been generating a useful `metrics.csv` in the lightning log. It's now generating a `tfevents` file instead without any loss data.

## To Reproduce
I'm not sure exactly when this change of behaviour occurred as I don't review the log after every run. To reproduce, I've run the GluonTS toy example which is outputting the `tfevents` file.

```python
import pandas as pd

from gluonts.dataset.pandas import PandasDataset
from gluonts.dataset.split import split
from gluonts.torch import DeepAREstimator

df = pd.read_csv(
    "https://raw.githubusercontent.com/AileenNielsen/"
    "TimeSeriesAnalysisWithPython/master/data/AirPassengers.csv",
    index_col=0,
    parse_dates=True,
)
dataset = PandasDataset(df, target="#Passengers")

training_data, test_gen = split(dataset, offset=-36)
test_data = test_gen.generate_instances(prediction_length=12, windows=3)

model = DeepAREstimator(
    prediction_length=12, freq="M", trainer_kwargs={"max_epochs": 5}
).train(training_data)
```

## Error message or code output

```
GPU available: True (mps), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
/Users/carlgoodwin/Library/r-miniconda-arm64/envs/r-reticulate/lib/python3.10/site-packages/lightning/pytorch/trainer/configuration_validator.py:74: You defined a `validation_step` but have no `val_dataloader`. Skipping val loop.

  | Name  | Type        | Params | In sizes                                                   | Out sizes   
------------------------------------------------------------------------------------------------------------------
0 | model | DeepARModel | 23.3 K | [[1, 1], [1, 1], [1, 48, 2], [1, 48], [1, 48], [1, 12, 2]] | [1, 100, 12]
------------------------------------------------------------------------------------------------------------------
23.3 K    Trainable params
0         Non-trainable params
23.3 K    Total params
0.093     Total estimated model params size (MB)
</span><span>Epoch 0: |          | 50/? [00:01&lt;00:00, 29.18it/s, v_num=149, train_loss=5.310]</span><span class="ace_constant ace_language" style="color: rgb(225, 197, 130);">Epoch 0, global step 50: 'train_loss' reached 5.31070 (best 5.31070), saving model to '/Users/carlgoodwin/R Projects/jump/lightning_logs/version_149/checkpoints/epoch=0-step=50.ckpt' as top 1
</span><span>Epoch 1: |          | 50/? [00:01&lt;00:00, 30.89it/s, v_num=149, train_loss=4.510]</span><span class="ace_constant ace_language" style="color: rgb(225, 197, 130);">Epoch 1, global step 100: 'train_loss' reached 4.50902 (best 4.50902), saving model to '/Users/carlgoodwin/R Projects/jump/lightning_logs/version_149/checkpoints/epoch=1-step=100.ckpt' as top 1
</span><span>Epoch 2: |          | 50/? [00:01&lt;00:00, 30.81it/s, v_num=149, train_loss=4.280]</span><span class="ace_constant ace_language" style="color: rgb(225, 197, 130);">Epoch 2, global step 150: 'train_loss' reached 4.28374 (best 4.28374), saving model to '/Users/carlgoodwin/R Projects/jump/lightning_logs/version_149/checkpoints/epoch=2-step=150.ckpt' as top 1
</span><span>Epoch 3: |          | 50/? [00:01&lt;00:00, 31.37it/s, v_num=149, train_loss=4.140]</span><span class="ace_constant ace_language" style="color: rgb(225, 197, 130);">Epoch 3, global step 200: 'train_loss' reached 4.14352 (best 4.14352), saving model to '/Users/carlgoodwin/R Projects/jump/lightning_logs/version_149/checkpoints/epoch=3-step=200.ckpt' as top 1
</span><span>Epoch 4: |          | 50/? [00:01&lt;00:00, 31.69it/s, v_num=149, train_loss=4.020]</span><span class="ace_constant ace_language" style="color: rgb(225, 197, 130);">Epoch 4, global step 250: 'train_loss' reached 4.01542 (best 4.01542), saving model to '/Users/carlgoodwin/R Projects/jump/lightning_logs/version_149/checkpoints/epoch=4-step=250.ckpt' as top 1
`Trainer.fit` stopped: `max_epochs=5` reached.
</span><span>Epoch 4: |          | 50/? [00:01&lt;00:00, 31.44it/s, v_num=149, train_loss=4.020]
```
I'm getting a `tfevents` file with no loss data ...
<img width="315" alt="image" src="https://github.com/awslabs/gluonts/assets/32419319/dfe07363-9d04-4f82-83cc-323d22b65f12">

... where I was previously getting a `metrics.csv` file with loss data.
<img width="315" alt="image" src="https://github.com/awslabs/gluonts/assets/32419319/3db6156d-e57b-4295-85dd-086dc753bff7">

I tried re-installing GluonTS and lightning.

## Environment
- Operating system: macOS sonoma 14.1.2
- Python version: 3.10.12
- GluonTS version: 0.14.3
- lightning version: 2.1.2

**Comment by lostella** [comment]
@cgoo4 estimators based on PyTorch use Lightning for training: the documentation for the latest version (https://lightning.ai/docs/pytorch/stable/extensions/logging.html) mentions the TensorboardLogger as being used by default. If I’m not wrong, older versions would use that if Tensorboard was installed, and the CVSLogger otherwise.

Things you could try:
- uninstalling Tensorboard from your environment
- explicitly specifying the CSVLogger as indicated in the docs above, via the `trainer_kwargs` to the gluonts estimator constructor.

(Removing the bug label)

**Comment by cgoo4** [comment]
@lostella thank you for the detective work! The below code does bring back the `metrics.csv`. I must have installed Tensorboard as a dependency to something.

```
from lightning.pytorch.loggers import CSVLogger

logger = CSVLogger("logs")

trainer_kwargs = {"logger": logger}
```
